addons/website_event_meet/static/src/js/website_event_meeting_room.js

```diff
@@ -0,0 +1,207 @@
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+      * Jitsi do not provide an REST API to get the number of participant in a room.
+      * The only way to get the number of participant is to be in the room and to use
+      * the Javascript API. So, to update the participant count on the server side,
+      * the participant have to send the count in RPC...
+      *
+      * When leaving a room, the event "participantLeft" is called for the current user
+      * once per participant in the room (like if all other participants were leaving the
+      * room and then the current user himself).
+      *
+      * "participantLeft" is called only one time for the other participant who are still
+      * in the room.
+      *
+      * We can not ask the user who is leaving the room to update the participant count
+      * because user might close their browser tab without hanging up (and so without
+      * triggering the event "videoConferenceLeft"). So, we wait for a moment (because the
+      * event "participantLeft" is called many time for the participant who is leaving)
+      * and the first participant send the new participant count (so we avoid spamming the
+      * server with HTTP requests).
+      *
+      * We use "setTimout" to send maximum one HTTP request per interval, even if multiple
+      * participants join/leave at the same time in the defined interval.
+      *
+      * Update on the 29 June 2020
+      */
+    _joinJitsiRoom: async function ($jitsiModal) {
+        await navigator.permissions.query({name: 'microphone'});
+        await navigator.permissions.query({name: 'camera'});
+
+        const domain = 'meet.jit.si';
+        const options = {
+            roomName: this.jitsiCode,
+            width: '100%',
+            height: '100%',
+            parentNode: $jitsiModal.find('.modal-body')[0],
+            configOverwrite: {disableDeepLinking: true},
+        };
+        let jitsiRoom = new window.JitsiMeetExternalAPI(domain, options);
+
+        let timeoutCall = null;
+        const updateParticipantCount = (joined) => {
+            // we clear the old timeout to be sure to call it only once each 2 seconds
+            // (so if 2 participants join/leave in this interval, we will perform only
+            // one HTTP request for both).
+            clearTimeout(timeoutCall);
+            timeoutCall = setTimeout(() => {
+                this.allParticipantIds = Object.keys(jitsiRoom._participants).sort();
+                if (this.participantId === this.allParticipantIds[0]) {
+                    // only the first participant of the room send the new participant
+                    // count so we avoid to send to many HTTP requests
+                    this._updateParticipantCount(this.allParticipantIds.length, joined);
+                }
+            }, 2000);
```

Comment:
we update every 2 seconds ? Seems a bit overkill

Reply:
@dbeguin It wait 2 seconds before updating (it will not update if nobody join/leave, and it will perform maximum 1 HTTP request each 2 seconds if people join/leave many times, during a long moment).

But we can increase this number for performance, I don't really know which timeout will be the best compromise between "fresh update" and "performance"